packages/tsu/build.sh

```diff
@@ -1,40 +1,21 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/cswl/tsu
 TERMUX_PKG_DESCRIPTION="A su wrapper for Termux"
 TERMUX_PKG_LICENSE="ISC"
```

Comment:
License should be fixed.
Ref: https://github.com/cswl/tsu/blob/v3.x/pip-wheel-metadata/tsu.dist-info/METADATA#L10

Reply:
Ohh.. didnt catch that..  the shell script was ISC..

The new one is MIT